dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/ZonedInterval.java

```diff
@@ -82,4 +82,12 @@ public ZonedInterval span(@Nonnull ZonedInterval interval) {
   public String toString() {
     return "[" + getStart() + ".." + getEndExclusive() + ")";
   }
+
+  @Override
+  public boolean equals(Object obj) {
+    if (!(obj instanceof ZonedInterval)) return false;
+    ZonedInterval interval = (ZonedInterval) obj;
+    return interval.getStart().isEqual(getStart())
```

Comment:
nit: In the same class the fields can be accessed directly, i.e. `interval.start.isEqualTo(start)`.